pymc_marketing/mmm/data_generator.py

```diff
@@ -0,0 +1,521 @@
+from pymc_marketing.mmm.transformers import (
+    delayed_adstock,
+    geometric_adstock,
+    logistic_saturation,
+    tanh_saturation)
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+import seaborn as sns
+import warnings
+
+warnings.filterwarnings('ignore', category=FutureWarning)
```

Comment:
What are these coming from?

Reply:
```
python3.11/site-packages/seaborn/_core.py:1218: FutureWarning: is_categorical_dtype is deprecated and will be removed in a future version. Use isinstance(dtype, CategoricalDtype) instead
  if pd.api.types.is_categorical_dtype(vector):
```

moved it back to matplotlib

Reply:
Nevermind, too many plots in other areas, just going to keep the warnings statement unless it's a big deal.

Reply:
They're coming from seaborn^